dbt/include/databricks/macros/relations/constraints.sql

```diff
@@ -79,8 +79,7 @@
 
 {% macro get_constraints_sql(relation, constraints, model, column={}) %}
   {% set statements = [] %}
-  -- Hack so that not null constraints will be applied before primary key constraints
-  {% for constraint in constraints|sort(attribute='type') %}
```

Comment:
And you've tested that we don't need to order them after your change?

Reply:
Not quite yet. But: the only two supported constraints are check and not_null. The order in that should not matter.
Will also see if I can test this.

Reply:
I was wrong to assume this hack was for pk/not_null only. In the meantime I've been able to get the integration-tests working, so it _should_ be fine now.